packages/ai/src/ai/account/models.py

```
@@ -0,0 +1,142 @@
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# =============================================================================
+# ACCOUNT MODELS
+# Shared internal types used by both the account layer and the task layer.
+# Placed here to avoid circular imports between account/auth and modules/task.
+# =============================================================================
+
+from pydantic import BaseModel
+from typing import List
```

Ruff (0.15.10)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)